servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategies.java

```diff
@@ -51,20 +51,33 @@ public static HttpExecutionStrategy defaultStrategy() {
      * paths. When merged with another execution strategy the result is always this strategy.
      *
      * @return {@link HttpExecutionStrategy} that disables all request-response path offloads.
-     * @see #anyStrategy()
+     * @see #offloadNone()
+     * @deprecated Replaced with more descriptive {@link #offloadNever()}.
      */
-    public static HttpExecutionStrategy noOffloadsStrategy() {
+    @Deprecated
+    public static HttpExecutionStrategy noOffloadStrategy() {
+        return OFFLOAD_NEVER_STRATEGY;
+    }
+
+    /**
+     * A special {@link HttpExecutionStrategy} that disables all offloads on the request-response and transport event
+     * paths. When merged with another execution strategy the result is always this strategy.
+     *
+     * @return {@link HttpExecutionStrategy} that disables all request-response path offloads.
+     * @see #offloadNone()
+     */
+    public static HttpExecutionStrategy offloadNever() {
         return OFFLOAD_NEVER_STRATEGY;
     }
 
     /**
      * An {@link HttpExecutionStrategy} that requires no offloads on the request-response path or transport event path.
-     * Unlike {@link #noOffloadsStrategy()}, this strategy merges normally with other execution strategy instances.
+     * Unlike {@link #offloadNever()}, this strategy merges normally with other execution strategy instances.
```

Comment:
Consider clarifying that it's expected to be returned from `requireOffloads()` method when a filter does not block.